scheduler/src/cook/mesos/api.clj

```diff
@@ -243,7 +241,7 @@
                   :priority (or priority util/default-job-priority)
                   :max-retries max_retries
                   :max-runtime (or max_runtime Long/MAX_VALUE)
-                  :ports (or ports [])
+                  :ports (or ports 0)
```

Comment:
What happens to clusters that previously used a list of ports? Will upgrading to fenzo cause db issues?